### PR TITLE
Add websocket to speak to api-server

### PR DESCRIPTION
**Pull request by fyndalf**

### CODE REVIEW

File: Node/files/node.py [hunk 1]
```diff
@@ -98,7 +99,14 @@ def get_localhost_of_host():
 def send_data(node_data):
     try:
         hosts_localhost = get_localhost_of_host()
-        requests.post(hosts_localhost, json=node_data, headers={'content-type':'application/json'}, timeout = 1)
+        #requests.post(hosts_localhost, json=node_data, headers={'content-type':'application/json'}, timeout = 1)
```
Comment (simsieg) [inline Node/files/node.py]: Remove this unneeded code

File: Node/Dockerfile [hunk 2]
```diff
@@ -2,6 +2,7 @@ FROM ethereum/client-go
 
 RUN apk add --update git bash python3 musl-dev gcc python3-dev py3-netifaces
 RUN pip3 install web3
+RUN pip3 install websocket-client
```
Comment (simsieg) [inline Node/Dockerfile]: You can do this in one pip3 call, so only one pip command and one temp docker-image needs to be created:
`RUN pip3 install web3 websocket-client`

File: Node/files/node.py [hunk 3]
```diff
@@ -64,7 +65,7 @@ def provide_data_every(n_seconds, web3):
         last_block_number = new_last_block_number
         node_data = gather_data(blocks_to_send, last_sent_block, web3)
         print(node_data)
-        send_data(node_data)
+        send_data(node_data)
```
Comment (simsieg) [inline Node/files/node.py]: No need to do that :D

File: Node/files/node.py (hunk 1 above)
Comment (fyndalf) [reply Node/files/node.py]: Already done in e2bda94 - however, I left the get_localhost_of_host()-call as outcommented code, as we might get it to work later on.

**Comment by fyndalf** [comment]
@simsieg How reliable is feram regarding python and the proposed fixes? Should they be accepted?

File: Node/files/node.py [hunk 4]
```diff
@@ -97,8 +98,14 @@ def get_localhost_of_host():
 
 def send_data(node_data):
     try:
-        hosts_localhost = get_localhost_of_host()
-        requests.post(hosts_localhost, json=node_data, headers={'content-type':'application/json'}, timeout = 1)
+        #hosts_localhost = get_localhost_of_host()
```
Comment (tom-lichtenstein) [inline Node/files/node.py]: Delete useless comment.

File: Node/files/node.py [hunk 5]
```diff
@@ -97,8 +98,14 @@ def get_localhost_of_host():
...
+        ws = create_connection("ws://172.18.0.1:3030")
```
Comment (tom-lichtenstein) [inline Node/files/node.py]: Transfer IP and port into a configuration. Try to get rid of the hardcoded IP.

**Comment by simsieg** [comment]
After fixing of Toms review, it will look good for me!

**Pull request closed without merging**